personalize.module

```diff
@@ -327,6 +328,36 @@ function personalize_pause_agents() {
 }
 
 /**
+ * Starts any agent that is due to be started.
+ *
+ * @return array
+ *   An array of two keys:
+ *   - An array of agents that were started
+ *   - An array of agents that were scheduled to start but could not be
+ *     started.
+ */
+function personalize_start_agents() {
+  $started = $error = array();
+  $all_agents = personalize_agent_load_multiple();
+  foreach ($all_agents as $agent_name => $agent) {
+    $status = personalize_agent_get_status($agent_name);
+    if ($status != PERSONALIZE_STATUS_RUNNING && $status !== PERSONALIZE_STATUS_COMPLETED) {
```

Comment:
Would be more future-proof to check that the status *is* one of "not started" or "paused".